src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --colorBlye: #00acdc;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("/fonts/Roboto-Medium.ttf");
+  font-weight: 500;
+  font-style: normal;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__link {
+  margin: 10px 0;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+}
+
+.nav__link {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  height: 60px;
```

Comment:
Please remove heightб because header height to big and if you remove it everything will be right
![image](https://user-images.githubusercontent.com/54936044/235102386-847ce122-ce5d-40a5-9ce2-15af804f60a0.png)
![image](https://user-images.githubusercontent.com/54936044/235102621-5443c050-1b3b-4216-837a-33d3db64265f.png)